type/shot.js

```diff
@@ -81,7 +82,8 @@ const ShotType = new GraphQLObjectType({
     tags: { type: new GraphQLList(GraphQLString) },
     user: { type: User },
     team: { type: User },
-    images: { type: Images }
+    images: { type: Images },
+    attachments: { type: Attachment }
```

Comment:
Yes, because attachments field should be a `new GraphQLList(Attachment)` and have to resolve by fetch attachments from dribbble API.

I think we can remove this line for now, I will create separate issue for shot's attachments and resolve function.

Or if you would like to work on this, you can see how `comments` field was resolved in master for example.

https://github.com/n3tr/dribbble-graphql/blob/master/type/shot.js#L84-L86

Thanks :)